modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/package.scala

```diff
@@ -15,6 +15,7 @@ package engine {
   object HandleP {
     def fromProcess(p: Process[Task, Event]): HandleP[Unit] = HandleP(Applicative[Handle].pure[(Unit, Option[Process[Task, Event]])](((), Some(p))))
   }
+  case class ActionMetadata(conditions: Conditions, operator: Option[Operator], observer: Option[Observer])
```

Comment:
`ActionMetadata` only exists as a way to pass parameters to Action. Its contents come from two places in the engine state, the global state, and the sequence metadata. Its life is short.